static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -40,6 +39,19 @@ const minArticlesBeforeShowingBanner = 3;
 
 const lastClosedAtKey = 'engagementBannerLastClosedAt';
 
+const getTestAndVariant = (): {
```

Comment:
just a bit more concise at the call site than having separate `getUserTest` and `getUserVariant` functions, since they were often called together